package.json

```diff
@@ -52,6 +52,7 @@
     "retext": "^5.0.0",
     "retext-smartypants": "^3.0.1",
     "snyk": "^1.88.1",
+    "tmp-promise": "^1.0.5",
```

Comment:
`tmp-promise` is not available on the docker image. suggest to use [tmp](https://www.npmjs.com/package/tmp)

Reply:
The docker image will pull in pipeline and its transitive dependencies, so that won't be an issue.